src/main/kotlin/javabot/operations/browse/Result.kt

```diff
@@ -0,0 +1,33 @@
+package javabot.operations.browse
+
+data class BrowseResult(
+        val artifactId: String = "none",
+        val binding: String = "none",
+        val uri: String = "none",
+        val components: IntArray = IntArray(0),
+        val match: IntArray = IntArray(0)
+) {
+    override fun equals(other: Any?): Boolean {
```

Comment:
why have you manually defined these two methods on a data class?

Reply:
Because of the arrays, which change the default behavior for equality and hashcodes. Practically, the fields aren't being returned by the source browser service, so they shouldn't affect the caching. I'm removing them.